src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -206,7 +206,9 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
     return trackConstraints;
   }
 
-  async checkContentShareConnectivity(): Promise<CheckContentShareConnectivityFeedback> {
+  async checkContentShareConnectivity(
+    sourceId: string = null
```

Comment:
Could we do sourceId? similar to startContentShareScreenCapture

Reply:
We are doing that in [MeetingReadinessChecker.ts](https://github.com/aws/amazon-chime-sdk-js/pull/687/commits/61a79a42f88fe7e900932dc4ff8fce3f5009be0b#diff-9d1c496522a94ecd712beccbc3e7d25bR47). In the implementation, we are setting it to `null` by default.